src/js/controller/model.js

```diff
@@ -111,6 +113,7 @@ define([
                             this.playVideo();
                         }, this);
                     }
+                    _provider.setPlaybackRate(this.get('defaultPlaybackRate'));
```

Comment:
You should use `model.setPlaybackRate` in case the streamType has already changed to "LIVE".